packages/cloudvision-grpc-web/src/grpc/GrpcSubjectWrapper.ts

```diff
@@ -0,0 +1,37 @@
+import { Observable, Subject, Subscription } from 'rxjs';
+
+export default class GrpcSubjectWrapper<T> extends Observable<T> {
```

Comment:
I don't believe the Observable class should ever be extended.
You should use the `new Observable(mySubscriber)` pattern